Protos/V1/task_status.proto

```diff
@@ -22,4 +22,5 @@ enum TaskStatus {
   TASK_STATUS_CANCELLED = 8; /** Task is cancelled. */
   TASK_STATUS_PROCESSING = 9; /** Task is being processed. */
   TASK_STATUS_PROCESSED = 10; /** Task is processed. */
+  TASK_STATUS_RETRIED = 11; /** Task is retried. */
```

Comment:
This is not a real status for now. What would be the use for such a status? What semantics to we want for it?

Reply:
This status would help distinguish tasks that are in error from too much retries (error) from task in error but retried and completed.
So retried tasks will be in retried status and last try would either be completed or error.